plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotColumnHandle.java

```diff
@@ -128,9 +129,12 @@ public static Type getTrinoTypeFromPinotType(FieldSpec.DataType dataType)
             case LONG:
                 return BigintType.BIGINT;
             case STRING:
+            case JSON:
                 return VarcharType.VARCHAR;
             case BYTES:
                 return VarbinaryType.VARBINARY;
+            case TIMESTAMP:
+                return TimestampType.TIMESTAMP_MILLIS;
```

Comment:
not sure if this is correct. `TimestampType.TIMESTAMP` is deprecated and set to `TimestampType.TIMESTAMP_MILLIS`

Reply:
`TimestampType.TIMESTAMP_MILLIS` is correct.